Cargo.lock

```diff
@@ -1074,9 +1074,9 @@ dependencies = [
 
 [[package]]
 name = "glean-ffi"
-version = "32.3.0"
+version = "32.4.0"
```

Comment:
Semver says these should be safe, but do we have anything to worry about with these being updated without a bump in the glean submodule? /cc @badboy